sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/SymbolicExecution/ExplodedGraphTests.cs

```diff
@@ -935,5 +935,93 @@ public void Test(string s)
 
             explodedGraph.Walk();
         }
+
+        [TestMethod]
+        [TestCategory("Symbolic execution")]
+        public void ExplodedGraph_DefaultLiteral()
+        {
+            var testInput = "var i = default(int); int j = default; System.IO.File k = default;";
```

Comment:
pedantic: can be a constant

Reply:
Indeed, but since all other `testInput` variables in this test file are declared the same way, I ll leave it as is for consistency